index.html

```diff
@@ -66,6 +66,285 @@ <h1 class="primary-text">I’m Abraham <br>Glad to see <br>you!</h1>
       </div>
     </div>
     <!-- intro end  -->
+
+    <div class="workDivider"></div>
+
+    <div class="main-workContainer">
+
+      <!-- portfolio section -->
+      
+      <section class="work-container">
+        <div class="project-img">
+          <img class='projesct-one-img' src="./assets/background-img/Snapshoot-Portfolio.svg" alt="work-background">
+        </div>
+
+        <!-- projectone-header -->
+        <div class="projectone-header">
+          <h2>Tonic</h2>
+        </div>
+
+        <div class='project-details-container'>
+          <ul class="project-details">
+            <li>
+              <h4 class="projectHeading">CANOPY</h4>
+            </li>
+            <li><img src="./assets/icons/Counter.svg" alt="counter"></li>
+            <li>
+              <p>Back End Dev</p>
+            </li>
+            <li><img src="./assets/icons/Counter.svg" alt="counter"></li>
+            <li>
+              <p>2015</p>
+            </li>
+          </ul>
+        </div>
+        <div class="project-text">
+          <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+        </div>
+
+        <!-- programming language used -->
+        <ul class="project-program-laguages">
+          <li class="program-language">
+            <p>html</p>
+          </li>
+          <li class="program-language">
+            <p>css</p>
+          </li>
+          <li class="program-language-javascript">
+            <p>javascript</p>
+          </li>
+        </ul>
+
+        <!-- project call to action section -->
+        <button type="submit" class="call-to-action">See project</button>
+      </section>
+      <!-- project one end -->
+
+      <div class="workDivider"></div>
+
+      <!-- project two start -->
+      <section class="work-container">
```

Comment:
- [ ] [REQUIRED] Good job so far! You've done a great job so far 👍🏼, but your **work** `cards` blend into the background and there's almost no spacing between the cards. Please review the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=848ucIcBgTd3IXHT-0) and try to style each of your cards like [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=848ucIcBgTd3IXHT-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/212110417-a99c153d-4409-4bab-af71-176a6274237e.png)

Reply:
thanks  done